src/AmsConfiguration.cpp

```diff
@@ -661,7 +850,20 @@ bool AmsConfiguration::save() {
 	address += saveBool(address, debugTelnet);
 	address += saveBool(address, debugSerial);
 	address += saveInt(address, debugLevel);
-
+//
+//  Domoticz 
+//
+	if(domoELIDX) {
+		address += saveBool(address, true);
+		address += saveInt(address, domoELIDX);	
+		address += saveInt(address, domoVL1IDX);	
+		address += saveInt(address, domoVL2IDX);
+		address += saveInt(address, domoVL3IDX);
+		address += saveInt(address, domoCL1IDX);
+		//address += saveString(address, String(domoEnergy).c_str());	
```

Comment:
If this is a unused configuration option, please remove

Reply:
yes. leftover from testing.

Reply:
done